styles/variables-base.less

```diff
@@ -396,28 +396,32 @@
 @agate-item-height:                   60px;
 @agate-item-height-large:             72px;
 @agate-item-indent:                   36px;
-@agate-item-inline-content-max-width: 240px;
+@agate-item-inline-content-max-width: 420px;
```

Comment:
add this new value only for Silicon

Reply:
This is needed for all skins because currently the layout is broken for inline Item
![broken](https://user-images.githubusercontent.com/63335823/111497195-95c4d800-8749-11eb-9412-6b465a71cba2.PNG)